### ISSUE TITLE
Smart DNS Proxy

### ISSUE THREAD

**Issue by YBS-PC** [AdguardTeam/AdGuardHome#5499]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### Description

is it possible to add smart dns proxy function for geo unblocking without vpn?  

If there is no such possibility in agh, tell me please where i can find a manual for creating smart dns on vps

**Comment by ainar-g** [comment]
Hello.  I'm not sure what you mean by “smart DNS”, but GeoIP ans AS number support is #1487.

**Comment by YBS-PC** [comment]
https://github.com/ab77/netflix-proxy

**Comment by ainar-g** [comment]
This seems like a combination of GeoIP detection with DNS rewrites (see `dnsrewrite` rules), perhaps with custom ECS  (see #1472).  Unless there's more to it, I feel like this issue is a duplicate of these.

**Comment by YBS-PC** [comment]
I think this is not the case at all. SmartDNS proxy is a DNS service that, in addition to the standard DNS function, performs additional functions, such as substituting part of the DNS query. With the help of this substitution, the sites are unblocked. SmartDNS is used as a solution for accessing blocked sites, i.e. those sites that are only available for certain countries and are not blocked at the provider level. Smart DNS unblocks Geo-Resticted Services
